generate-db-diagram/action.yml

```diff
@@ -23,7 +23,7 @@ inputs:
 runs:
   using: composite
   steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
```

Comment:
@quesabe what about other actions and errors related to things like `set-output` (in `set-commit-hash` action)?

Reply:
I've added a description in the ticket. The main idea is that the remaining warnings come from the actions we use and won't go until the maintainers update the actions.

> The list of actions still causing the warnings:
> 1. latest-release-commit-hash:
> - InsonusK/get-latest-release (Node12 warning, an issue is open);
> 2. create-release:
> - softprops/action-gh-release (Node12 warning, an issue is open);
> - dev-drprasad/delete-tag-and-release (Node12 warning, an issue is open; this one is used in action testing and does not affect our own actions).
>
> With the action we depend on their maintainers and are not currently ready for update.